### PR TITLE
Add execution result

### PR DESCRIPTION
**Pull request by MoshiMoshi0**
Support for storing execution result as json in database.

Preview:
lis
fai
suc

### CODE REVIEW

File: simple_scheduler/jobs/sample_job.py [hunk 1]
```diff
@@ -22,6 +22,7 @@ def meta_info(cls):
 
     def run(self, argument1, argument2, *args, **kwargs):
         print('Hello from AwesomeJob! Argument1: %s, Argument2: %s' % (argument1, argument2))
+        return [argument1, argument2]
```
Comment (wenbinf) [inline simple_scheduler/jobs/sample_job.py:25]: Could you also change the block comment here https://github.com/Nextdoor/ndscheduler/blob/master/ndscheduler/job.py#L116

Make it clear that the return value of this function will be stored in database and be shown as result in UI.

**Review by wenbinf — COMMENTED** [review]
I ran your code on my local dev, which worked very well! Thanks again for the change.

Could you address my comment below and make sure "make flake8" pass without error (so the CI can pass)?

**Comment by MoshiMoshi0** [comment]
Updated the docstring, hope it's informative enough.

The only flake8 errors are in code that I did not write or change:

```
simple_scheduler/scheduler.py:18:1: E305 expected 2 blank lines after class or function definition, found 1
simple_scheduler/jobs/slack_job.py:76:1: E305 expected 2 blank lines after class or function definition, found 1
```

I don't know if you want me to fix those in this PR.

File: ndscheduler/job.py [hunk 2]
```diff
@@ -48,7 +48,7 @@ def get_failed_description(cls):
         return 'hostname: %s | pid: %s' % (hostname, pid)
 
     @classmethod
-    def get_succeeded_description(cls):
+    def get_succeeded_description(cls, result=None):
```
Comment (devcompl) [inline ndscheduler/job.py]: Don't know if the default value is needed here as we are always passing the result to this method.

Comment (MoshiMoshi0) [inline ndscheduler/job.py]: Don't know if the default value is needed here as we are always passing the result to this method.

**Review by wenbinf — APPROVED** [review]
If it's just two-line changes to fix 'make flake8' error, could you do that in this PR? so the CI can turn green :)

Otherwise, this PR is good to go.

**Comment by wenbinf** [comment]
@swaggy could you help merge this PR? I don't have write access to this repo after I left Nextdoor :(

**Pull request merged**